konrad/convection.py

```diff
@@ -280,8 +281,8 @@ def test_profile(self, T_rad, p, phlev, surface, surfaceT, lp,
 
         eff_Cp_s = surface.heat_capacity
 
-        diff = energy_difference(T_con, T_rad, surfaceT, surface.temperature,
-                                 dp, eff_Cp_s)
+        diff = energy_difference(T_con, T_rad, surfaceT,
+                                 surface['temperature'], dp, eff_Cp_s)
```

Comment:
Both lines did exactly the same as `surface['temperature']` is an one-dimensional array 😉 but I fixed it anyhow to prevent confusion in the future. Thanks for the catch 👍